### PR TITLE
Npx & apirule fix

### PR DESCRIPTION
**Pull request by Wawrzyn321**
**Description**

Changes proposed in this pull request:

- Simplify auth process in pamela
- Add instructions for building & running Console npx
- Fix CMF resolving
- Fix `domain` resolving in apirules - get one from `gateway` instead from URL iff k8sApiUrl is missing

**Related issue(s)**

### CODE REVIEW

File: core-ui/src/components/ApiRules/useGetGatewayDomain/useGetGatewayDomain.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { useGet, useMicrofrontendContext } from 'react-shared';
+
+const apiGatewayUri =
+  '/apis/networking.istio.io/v1beta1/namespaces/kyma-system/gateways/kyma-gateway';
+
+export function useGetGatewayDomain() {
+  const k8sApiUrl = useMicrofrontendContext().k8sApiUrl;
```
Comment (dariadomagala) [inline core-ui/src/components/ApiRules/useGetGatewayDomain/useGetGatewayDomain.js:8]: do we need to additionally make the call for the gatewayURI later on if we already have the domain?

Comment (Wawrzyn321) [reply core-ui/src/components/ApiRules/useGetGatewayDomain/useGetGatewayDomain.js:8]: We make no call if the domain is already here - `skip: !!domain,`

Comment (dariadomagala) [reply core-ui/src/components/ApiRules/useGetGatewayDomain/useGetGatewayDomain.js:8]: right :)

**Review by dariadomagala — APPROVED** [review]

**Review by valentinvieriu — APPROVED** [review]

**Pull request merged**